### DISCUSSION TITLE
laser centering

### DISCUSSION THREAD

**Discussion by lanel8** [arkypita/LaserGRBL#1232] [Q&A]
So I want to engrave names on wood coasters. What I do is center the laser on the center of my project and mark it as the zero point, then I upload my name file and edit the size then before I start  engraving, I check the boundary. Every time I check it. the laser runs off the project and goes to the right corner a couple of inches even though I set the project's center as the new zero point. Any suggestions?

**Comment by srdjan93s** [comment]
Hello,
after you center laser, do you confirm it with SetNewZero command (can be
found in Hotkeys window)?
So you get new coordinate system - machine and work one

суб, 26. дец 2020. у 23:13 lanel8 <notifications@github.com> је написао/ла:

**Comment by lanel8** [comment]
yes
